src/NUnitFramework/nunitlite.tests/TestUtilities/TestBuilder.cs

```diff
@@ -48,9 +48,7 @@ public static TestSuite MakeFixture(Type type)
 
         public static TestSuite MakeFixture(object fixture)
         {
-            TestSuite suite = MakeFixture(fixture.GetType());
-            suite.Fixture = fixture;
-            return suite;
+            return MakeFixture(fixture.GetType());
```

Comment:
So the question is whether all the tests that need to pre-create a fixture eventually go through RunTestSuite.